spec/intl.html

```diff
@@ -1389,7 +1389,7 @@ <h1>
           <h1>
             CalendarDateEra (
               _calendar_: a String,
-              _datetime_: an Object with the [[ISOYear]] internal slot,
+              _date_: an Object with the [[ISOYear]], [[ISOMonth]] and [[ISODay]] internal slots,
```

Comment:
```suggestion
              _date_: an Object with the [[ISOYear]], [[ISOMonth]], and [[ISODay]] internal slots,
```

same throughout

Reply:
I tried that, but it didn't seem to get rid of the build errors.

Reply:
Aaaah, hehe. Sure, I don't feel strongly about it.

Reply:
To fix the build errors I think this will have to be something like "a Temporal.PlainDate, Temporal.PlainYearMonth, Temporal.PlainMonthDay, or Temporal.PlainDateTime" (delete as appropriate, e.g. PlainMonthDay isn't passed to CalendarDateEra)